### ISSUE TITLE
Support Windows 10 OS Build 15063.674 / 16299.xxxxx

### ISSUE THREAD

**Issue by dummys** [hfiref0x/UPGDSED#10]
Hi,

This version  is RS2 right ? I'm wondering why I said that it's not supported.
Is it a bug or it's a minor revision of the RS2 update ?

**Comment by dummys** [comment]
Hi again,

Changing the `MAX_SUPPORTED_NT_BUILD` in `global.h` to `15063.674` seems to fix the problem.
No error in the log when patching.

**Comment by hfiref0x** [comment]
15063/14393 already supported. You installed update, MS doesn't automatically release new symbols for ntoskrnl with each monthly/weekly update. Usually it take them some time (few weeks at worst). This program relies on MS symbols to find offsets for patch. Changing decimal build number in header to be float also make no sense and won't compile.

It is already discussed 

here
https://github.com/hfiref0x/UPGDSED/issues/4

here
https://github.com/hfiref0x/UPGDSED/issues/3

and here
https://github.com/hfiref0x/UPGDSED/issues/7

No need to open new issue for each new Windows update.

**Comment by dummys** [comment]
It compils actually, you can test it. And I didnt get anymore the error "unsupported version".

**Comment by hfiref0x** [comment]
No it is not. Changing this value doesn't make it magically support unknown kernel.

**Comment by dummys** [comment]
The kernel is not unknown it's the same in version 15063. It's just the tool that check for it and sees 15063.674

**Comment by hfiref0x** [comment]
Show your initial error screen.

**Comment by dummys** [comment]
imgur.com/a/aAsTn

**Comment by hfiref0x** [comment]
You are patching Windows 7 RTM.

**Comment by dummys** [comment]
ugly patched: [patched](https://imgur.com/a/ZRlMj)

**Comment by hfiref0x** [comment]
Have no idea but program detect your initial kernel as Windows 7. Which mean you have ntoskrnl with Win7 resource inside %systemroot%\system

**Comment by dummys** [comment]
no it's windows 10 [win10](https://imgur.com/a/a3zeR)

**Comment by hfiref0x** [comment]
Do you run original program with Windows 7 RTM compatibility setting?

**Comment by dummys** [comment]
Yes, you are right, just checked and it was in compatibility mode. Here the new log: [patch_fail](https://imgur.com/a/P96U6)

**Comment by dummys** [comment]
Ho, I see. Perhaps the compiled version has more fix than the release one ?

**Comment by hfiref0x** [comment]
There was an issue with storing symbols in current process directory in v1.12, workaround was posted in dev branch here v1.13.

**Comment by dummys** [comment]
Yes but I downloaded the repo via zip method, and the site called UPGDSED-master, so fix is not inside.

**Comment by hfiref0x** [comment]
Do you "unblocked" exe after extracting from archive? File Properties->Unblock.

**Comment by dummys** [comment]
Yes, otherwise I can't launch it. I've just checked the commit for version 1.1.3 and I see that it's version 1.1.2 in the folder.

**Comment by dummys** [comment]
Ok I see what is different. In release mode, it's not working, in debug mode it's working.

**Comment by hfiref0x** [comment]
Leave this lines
        _strcpy(szBuffer, g_szTempDirectory);
        _strcat(szBuffer, NTOSKRNMP_EXE);

**Comment by dummys** [comment]
Yes, I've already done it like this. It was just the copy/past in github which I forget  to remove it.

**Comment by hfiref0x** [comment]
Do you run app from VS (both debug and release)? Maybe VS symbols loading settings somehow affect application.

**Comment by dummys** [comment]
Nop, I run it from elevated command prompt

**Comment by hfiref0x** [comment]
You can try track it with procmon (file/network).

**Comment by dummys** [comment]
weird issue then

**Comment by hfiref0x** [comment]
I will add AppCompat mode checking in next version so program will produce warning about it, can't say anything about this weird symbols issue as I cannot reproduce it right now.

**Comment by dummys** [comment]
Ok, I will check it tomorrow with procmon.

**Comment by oakgary** [comment]
do we have the same issue atm? 
symbols got released but the patcher still tells me unsupported version
https://s.put.re/Lg57sqo.PNG

**Comment by hfiref0x** [comment]
This version does not support RS3.

**Comment by RealKallos** [comment]
I look forward to RS3 being supported.

**Comment by hfiref0x** [comment]
It is in dev branch, https://github.com/hfiref0x/UPGDSED/tree/dev

**Comment by RealKallos** [comment]
yeah i see :) thx

**Comment by hfiref0x** [comment]
Version 1.2.0 with RS3 support merged from dev to master in 8ebb8402fcf006b01b8dc027745da159ecfaa5cc
